Tests/EventListener/SecurityListenerTest.php

```diff
@@ -26,20 +26,24 @@ class SecurityListenerTest extends \PHPUnit_Framework_TestCase
      */
     public function testLegacyAccessDenied()
     {
-        $this->iniSet('error_reporting', -1 & ~E_USER_DEPRECATED);
+        if (!interface_exists('Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface')) {
+            $this->markTestSkipped();
```

Comment:
This doesn't make any sense. The test precisely is there to test the old (deprecated) way of checking access. What is the actual issue you try to solve?